testing/constants.ts

```diff
@@ -1,3 +1,9 @@
 export enum HttpStatusCode {
   Unauthorized = 401,
 }
+
+// Note that this should be kept in sync with the "PacksAllowedPublicDomains" runtime config 
+export const DEFAULT_ALLOWED_GET_DOMAINS_REGEXES = [
+  /coda-us-west-2-[prod|staging|head|adhoc|dev]-blobs-upload.s3.us-west-2.amazonaws.com$/,
```

Comment:
Nit, let's just make the env part `\w+` so we don't advertise all the other environments.

Reply:
ah see this, will do